dhocs/src/Dhall/Dhocs.hs

```diff
@@ -0,0 +1,67 @@
+{-| This module contains the top level and options parsing of the @dhocs@ executable
+-}
+{-# LANGUAGE OverloadedStrings #-}
+
+module Dhall.Dhocs
+    ( -- * Options
+      Options(..)
+    , GenerationStrategy(..)
+    , parserInfoOptions
+    , parseOptions
+
+      -- * Execution
+    , main
+    ) where
+
+import Data.Semigroup      ((<>))
+import Options.Applicative
+
+
+-- | To specify if we should generate a single HTML page with all your package info
```

Comment:
Style nit: I think it's a bit unusual to address the user with `you` and `your` in the haddocks. I don't mind though. It's kind of nice too! :)

Reply:
Maybe I'll read more haddock dhocs to try to adhere with the current write style!

Reply:
It could change to something like
```suggestion
-- | To specify if we should generate a single HTML page with all the package information
```
I also changed info -> information, I like to keep the less abbreviations as I can. Yesterday writing the code in a kind of hurry I passed this

Reply:
On second thought, what seemed unusual wasn't that you're directly addressing the reader.

It was that this sentence seemed to assume that the reader and the user are the same person.

No worries though, this stuff is fairly unimportant I think.